.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on: [push, pull_request]
```

Comment:
```suggestion
on:
  push:
    branches:
      - main
      - master
  pull_request:
```
Yeah this will prevent running jobs 2x on PRs

Reply:
how does it doe this?